### ISSUE TITLE
Translation fixes 5 (ru)

### ISSUE THREAD

**Issue by Svetaleet** [TRSasasusu/TranslationForTheOutsider#31]
The last one, I hope. Correcting minor logical errors

[russian.txt](https://github.com/TRSasasusu/TranslationForTheOutsider/files/14236205/russian.txt)

**Comment by TRSasasusu** [comment]
Thank you for updating the translation text.

I would like to ask you that the previous translation of the ship log of the projection pool house is
>Дурман попросил Бадана передать Соланум, что ей не стоит торопиться, и передать Осмунде, что он сожалеет обо всём, и они скоро снова увидятся.

but, your latest one is
>Дурман попросил Бадана передать сообщение Осмунде и Соланум.

It seems get very shortened, so, is it really ok?

**Comment by Svetaleet** [comment]
Maybe I'm wrong, but I thought there was too much relationship description here for a ship log. I understand that they are in the English version, but the translation seems... weird? I'm certainly not a linguist, but the translation of this line specifically looks wrong. I tried to achieve a result similar in meaning to the original and looking normal, but I couldn't. So, I decided to shorten it to just "he sent a message". Still conveys the meaning. Perhaps it's the difficulty of translation. I really had a hard time describing the whole message in one sentence to make it look readable
